README.md

```diff
@@ -6,6 +6,110 @@
 
 Bivrost is in very early development. Do not use this for anything important.
 
+## Description
+
+`bivrost` generates Swift classes for Solidity contracts that know how to encode and decode their function calls. Use this to generate transactions that can then be sent to the blockchain.
+
+You can use this as a pre-compile script phase for your Swift project. Generate the needed files, then add them to your project.
+
+`bivrost` creates an output folder that contains auxiliary code for encoding and decoding as well as generated contract classes.
+
+## Usage
+
+### [Mint](https://github.com/yonaskolb/mint) 🌱
+
+    $ mint run gnosis/bivrost-swift "bivrost --input './contracts/*.json' --output '../Sources/Solidity'"
+
+### SPM
+
+    $ git clone https://github.com/gnosis/bivrost-swift
+    $ cd bivrost-swift
+    $ swift run bivrost --input './contracts/*.json' --output '../Sources/Solidity'
+
+Using `BivrostKit` from code is currently not supported, but is on the roadmap.
+
+## Options
+
+- `--input` [default: ./abi/*.json] - Input file pattern specifying which json files should be parsed. Needs to be escaped with quotes to prevent the shell from expanding it.
+- `--output` [default: ./solidity] - Output folder which will contain all necessary Solidity files (generated and auxiliary).
+- `--force` [default: false] - If set, deletes all contents of the output folder before recreating it. Make sure there is nothing else in there.
+
+Run `$ mint run gnosis/bivrost-swift "bivrost --help"` for current help.
+
+## Input files
+
+`bivrost` parses the Solidity contract JSON files for code generation. They should have the following format:
```

Comment:
You can add that it is the format used by the truffle compiler. We need to add that in our repo.

Reply:
Will do